### ISSUE TITLE
Unable to plan with a readonly api key since 1.2.0

### ISSUE THREAD

**Issue by lra** [mongodb/terraform-provider-mongodbatlas#659]
Hi,

We are using an "Organization Read Only" API key do terraform plans.
Since 1.2.0 version of the mongodbatlas provider, the API key returns this error:
```
│ Error: error getting project's api keys (REDACTED): GET https://cloud.mongodb.com/api/atlas/v1.0/groups/REDACTED/apiKeys: 401 (request "USER_UNAUTHORIZED") Current user is not authorized to perform this action.
│ 
│   with module.gnuat03.mongodbatlas_project.this,
│   on ../../modules/cms-atlas-database/project.tf line 16, in resource "mongodbatlas_project" "this":
│   16: resource "mongodbatlas_project" "this" {
```

No problem with 1.1.1.

The "Organization Read Only" API key is supposed to "Provides read-only access to everything in the organization, including all projects in the organization." so I'm not sure if that's a provider issue, or an Atlas API permission issue.

**Comment by themantissa** [comment]
@lra did you also reach out to MongoDB support today?

**Comment by lra** [comment]
The problem was acknowledged by the Atlas team, and should update this issue once the problem is solved.

**Comment by themantissa** [comment]
@lra correct, we will have the team investigate and address the issue.   Thank you!  Internal ticket: INTMDB-295

**Comment by themantissa** [comment]
The fix for this was just released in version 1.3.0.   Thanks!
